src/mono/browser/runtime/memory.ts

```diff
@@ -390,6 +390,13 @@ export function localHeapViewF64(): Float64Array {
     return Module.HEAPF64;
 }
 
+export function copyBytes(srcPtr: VoidPtr, dstPtr: VoidPtr, bytes: number): void {
+    const heap = localHeapViewU8();
+    const src = heap.subarray(srcPtr as any, srcPtr as any + bytes);
+    const dst = heap.subarray(dstPtr as any, dstPtr as any + bytes);
+    dst.set(src);
```

Comment:
It should be possible to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/copyWithin